pkg/dao/repository_config_test.go

```diff
@@ -1001,9 +1019,14 @@ func (suite *RepositoryConfigSuite) TestValidateParameters() {
 	assert.True(t, response.URL.MetadataPresent)
 	assert.False(t, response.URL.Skipped)
 
-	mockExtRDao.Mock.On("ValidRepoMD", "http://badrepo.com").Return(404, nil)
+	// Providing a bad url that doesn't have a repo
+	parameters = api.RepositoryValidationRequest{
+		Name: pointy.String("Some bad repo!"),
+		URL:  pointy.String("http://badrepo.com"),
+	}
+	mockExtRDao.Mock.On("FetchRepoMd", "http://badrepo.com").Return(pointy.String(""), 404, nil)
```

Comment:
```suggestion
	mockExtRDao.Mock.On("FetchRepoMd", "http://badrepo.example.com").Return(pointy.String(""), 404, nil)
```

https://www.rfc-editor.org/rfc/rfc2606.html#page-2